Sources/Client/Model/Channel/ChannelsQuery.swift

```diff
@@ -19,7 +19,7 @@ public struct ChannelsQuery: Encodable {
         case presence
         case pagination
         case messagesLimit = "message_limit"
-        case memberLimit = "member_limit"
+        case membersLimit = "member_limit"
```

Comment:
why this change when the field is named `"member_limit"`?

Reply:
To be consistent with other properties.